src/scancode/cache.py

```diff
@@ -41,9 +41,7 @@
 from commoncode.fileutils import path_to_unicode
 from commoncode.system import on_linux
 from commoncode import timeutils
-
-from scancode import scans_cache_dir
-
+from licensedcode import DEV_MODE
```

Comment:
I think this should be across all scrancode, not just licensedcode

Reply:
Agreed. I thought about what would be a better place for it, probably `commoncode`. But it seemed like moving a few of those globals would drag a whole lot of stuff with it, so I opted for keeping changes to a minimum.
I can try to move all globals to `commoncode/__init__.py` (and uppercase them while I'm at it). Should I?